controller/templates/statefulset.yaml

```diff
@@ -111,20 +111,32 @@ spec:
           # should be put into an init container.
           livenessProbe:
             httpGet:
+              # Required because rstudio sends a 404 even if everything is ok and the user-agent is left out.
+              httpHeaders:
+                - name: User-Agent
+                  value: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36"
```

Comment:
Do we have to be that specific or do you know if there is some kind of generic "I am a browser" value that Rstudio would accept

Reply:
Yeah I just took this from my browser requests. Let me see if there is a more basic version that rstudio will accept.